media/css/firefox/whatsnew/whatsnew-110-en.scss

```diff
@@ -0,0 +1,226 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
```

Comment:
Question (non-blocking): With the exception of `lib`, are these imports needed?

This is assuming we're also loading the style sheet for the default WNP, as well as this WNP) (see previous comment).